modules/CmpdRegBulkLoader/src/server/routes/CmpdRegBulkLoaderRoutes.coffee

```diff
@@ -18,6 +21,8 @@ exports.setupRoutes = (app, loginRoutes) ->
 	app.post '/api/cmpdRegBulkLoader/validateCmpds', loginRoutes.ensureAuthenticated, exports.validateCmpds
 	app.post '/api/cmpdRegBulkLoader/checkFileDependencies', loginRoutes.ensureAuthenticated, exports.checkFileDependencies
 	app.post '/api/cmpdRegBulkLoader/purgeFile', loginRoutes.ensureAuthenticated, exports.purgeFile
+	app.get '/api/cmpdRegBulkLoader/getSDFFromBulkFileId/:bulkFileID', loginRoutes.ensureAuthenticated, exports.getSDFFromBulkFileId
```

Comment:
nit: Elsewhere in CReg we call these files `BulkLoadFile`s rather than "bulkFile", so I think for consistency it'd be better to name these APIs and functions something like `getSDFFromBulkLoadFileId`, `exports.getSDFFromBulkLoadFileId`. It's a small point, but it helps prevent confusion (is a bulkFile different from a bulkLoadFile?) and helps if we ever do searches across the code or big find/replace operations for all usages of "bulkLoadFile".